nginx-nodejs-redis/web/server.js

```diff
@@ -0,0 +1,24 @@
+cost os = require('os');
+const express = require('express');
+const app = express();
+const redis = require('redis');
+const redisClient = redis.createClient({
+  host: 'redis',
+  port: 6379
+});
+
+app.get('/', function(req, res) {
+    redisClient.get('numVisits', function(err, numVisits) {
+        numVisitsToDisplay = parseInt(numVisits) + 1;
+        if (isNaN(numVisitsToDisplay)) {
+            numVisitsToDisplay = 1;
+        }
+        res.send('Number of visits is: ' + numVisitsToDisplay);
```

Comment:
You should add the `os.hostname()` if you want to display which node is responding
```suggestion
       res.send(os.hostname() +': Number of visits is: ' + numVisitsToDisplay);
```